pmd-apex/src/main/resources/category/apex/performance.xml

```diff
@@ -80,4 +80,26 @@ public class Something {
         </example>
     </rule>
 
+    <rule name="AvoidDatabaseMethodCallsInLoops"
+          language="apex"
+          since="6.29.0"
+          message="Avoid database method calls inside loops"
+          class="net.sourceforge.pmd.lang.apex.rule.performance.AvoidDatabaseMethodCallsInLoopsRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_apex_performance.html#avoiddatabasemethodcallsinloops">
```

Comment:
How can I test this link? 

I followed the instructions at https://pmd.github.io/latest/pmd_devdocs_writing_documentation.html#building but the Jekyll site only contained the sidebar links, all of the rules returned a 404.

Reply:
The rule documentation is generated by the module "pmd-doc". If you build this module, it will create the files under `docs/pages/pmd/rules` (these files are not committed).